src/stdio_mgr/stdio_mgr.py

```diff
@@ -37,9 +37,12 @@ class _PersistedBytesIO(BytesIO):
     """Class to persist the stream after close.
 
     A copy of the bytes value is available at :attr:`_closed_buf` after
-    the :meth:`~BytesIO.close`.
+    the :meth:`~io.IOBase.close`.
     """
 
+    #: |str| copy of the stream contents immediately before close
```

Comment:
@jayvdb This is the only (possibly) controversial change.

In order to provide a link target to point to for `_closed_buf`, it has to be defined on the class and provided with the special `#:` documentation comment.

This could be moved inside an `__init__`, which might be preferable.

```
class ...:
    ...

    def __init__(self):
        #: {documentation comment}
        self._closed_buf = None
```

Also, `None` might not be the best default, but I figure it's better than the empty string, b/c empty string is a possible valid value of the contents of `_closed_buf` after close.

Reply:
That was dodgy.  https://github.com/bskinn/stdio-mgr/pull/58 is how I would tidy it up